test/common_device_type.py

```diff
@@ -364,6 +391,32 @@ def multi_fn(slf, devices, *args, **kwargs):
         return multi_fn
 
 
+# Specifies per-dtype precision overrides.
+# Ex.
+#
+# @overridePrecision(torch.half : 1e-2, torch.float : 1e-4)
+# @dtypes(torch.half, torch.float, torch.double)
+# def test_X(self, device, dtype):
+#   ...
+#
+# When the test is instantiated its class's precision will be set to the
+# corresponding override, if it exists.
+# self.precision can be accessed directly, and it also controls the behavior of
+# functions like self.assertEqual().
+class overridePrecision(object):
```

Comment:
nit: I have a slight preference with starting the name with the most relevant thing.

The most relevant thing here is that I'm setting the precision, not that it's overriding some default (which is an implementation detail).

Reply:
@precisionOverride it is.